lib/Extension/Bundle/AbstractPimcoreBundle.php

```diff
@@ -90,4 +84,18 @@ public function getEditmodeCssPaths()
     {
         return [];
     }
+
+    public static function isInstalled(): bool
+    {
+        if (!self::$bundleManager) {
+            self::$bundleManager = \Pimcore::getContainer()->get(PimcoreBundleManager::class);
+        }
+
+        $bundle = self::$bundleManager->getActiveBundle(__CLASS__, false);
+        if (!self::$bundleManager->isInstalled($bundle)) {
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
I think we need late static binding here. In the trait it was fine, but we want the values from the subclass, not from `AbstractPimcoreBundle`, right?

```suggestion
    public static function isInstalled(): bool
    {
        static::$bundleManager ??= \Pimcore::getContainer()->get(PimcoreBundleManager::class);

        $bundle = static::$bundleManager->getActiveBundle(static::class, false);

        return static::$bundleManager->isInstalled($bundle);
    }
```